lib/mayaHydra/mayaPlugin/renderOverride.cpp

```diff
@@ -340,6 +341,15 @@ Ufe::Path usdPathToUfePath(
         registration->pluginSceneIndex, usdPath) : Ufe::Path();
 }
 
+inline bool areDifferentForOneOfTheseBits(unsigned int val1, unsigned int val2, unsigned int bitsToTest)
+{
+    if ((val1 & bitsToTest) != (val2 & bitsToTest)) {
+        return true;
+    }
+
+    return false;
```

Comment:
Nit-pick: could just be
```suggestion
    return  ((val1 & bitsToTest) != (val2 & bitsToTest));
```